packages/http-multipart-body-parser/index.js

```diff
@@ -2,6 +2,16 @@ const BusBoy = require('busboy')
 const contentTypeLib = require('content-type')
 const createError = require('http-errors')
 
+const sanitizeHeaders = (headers) => {
```

Comment:
this function will mutate input, where lowercase 'content-type' key header would be propagated to remaining middlewares and handler. As a user of `http-multipart-body-parser` middleware, I wouldn't expect that from the middleware. Can we change this function to just get `/content-type/i` property?

Reply:
I agree with this comment.

As a reference we are doing something similar to what @dkatavic is suggesting in other middlewares:

 - https://github.com/middyjs/middy/blob/1.0.0-beta/packages/http-content-negotiation/index.js#L28
 - https://github.com/middyjs/middy/blob/1.0.0-beta/packages/http-urlencode-body-parser/index.js#L13